### PR TITLE
some fixes to python example scripts for python3 compatibility

### PR DESCRIPTION
**Pull request by c-holtermann**

### CODE REVIEW

File: bindings/python/example_scripts/account_analysis.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def period_end(start_year, start_month, period_type):
     # so we get a period end like
     # 2010-03-31 for period starting 2010-01 instead of 2010-04-01
     return date(end_year, end_month, 1) - ONE_DAY
-    
+
 
 def generate_period_boundaries(start_year, start_month, period_type, periods):
     for i in xrange(periods):
```
Comment (bhardwajs) [inline bindings/python/example_scripts/account_analysis.py]: Shouldn't xrange be changed to range?

Comment (c-holtermann) [reply bindings/python/example_scripts/account_analysis.py]: Good point. I'm sure there are some more things like this.

**Comment by jralls** [comment]
Do you want to keep working on this PR until Friday (when I start putting the release together) or make a new one for more changes?

**Comment by c-holtermann** [comment]
Let's keep this one till friday.

**Comment by jralls** [comment]
Well, that Friday came and went along with the release of 3.3, so this missed it. I've merged now to keep that from happening again. You'll just have to make a new PR when you get around to working on this again.

Thanks.

**Comment by c-holtermann** [comment]
Thanks. I was busy with some other projects but will pick this up again the coming weeks. Good to be reminded.

**Comment by c-holtermann** [comment]
Until the next release things about python3 should be fixed (that's my goal). I saw this PR as part of that and didn't feel a need to hurry.

**Pull request merged**